### DISCUSSION TITLE
AdGuard Home и nfqws keenetic вопрос совместной работы

### DISCUSSION THREAD

**Discussion by pumi** [Anonym-tsk/nfqws-keenetic#60] [Q&A]
Возможно ли установить AdGuard Home и nfqws на keenetic совместно?

Ссылки инструкций на всякий случай по AdGuard  https://github.com/AdguardTeam/AdGuardHome и https://telegra.ph/AdGuard-Home-Keenetic-09-07
nfqws keenetic уже установлено на флешку и нормально работает.
Начал читать и немного запутался, особенно момент с DNS серверами:
по инструкции nfqws:
- Рекомендуется игнорировать предложенные провайдером адреса DNS-серверов.
- Вместе с этим рекомендуется [настроить использование DoT/DoH].
Всё это сделал и работает обход/замедление.
А при установке AdGuard Home написано:
- Отключите встроенный DNS-сервер прошивки Keenetic. 
- НЕ игнорировать DNSv4 интернет-провайдера

Как то не стыкуется. У меня сейчас выключены DNS провайдера, а несколько DoH подключено.
Что делать подскажите?

**Comment by Anonym-tsk** [comment]
Это лишь рекомендация. Если у вас работает с вашими днс, все ок

**Reply by pumi** [reply]
Я правильно понимаю, что в Кинетике удалить все прописанные мной DNS-серверы и вписать DNS-сервер AdGuard Home 192.168.1.1:53 а уже в настройках AdGuard Home прописать в upstream DNS любые сторонние DNS серверы?

**Comment by H4X3** [comment]
Кто-то разобрался в теме, как правильно поставить AdGuard Home и nfqws?

**Comment by LimpingK** [comment]
> У меня сейчас выключены DNS провайдера, а несколько DoH подключено.

К слову, у меня - так же.
И я не хочу включать единственный AdGuard: при блокировках то один сервис DNS не работает, то другой. Когда на кинетике добрый десяток DoH, он выбирает лучший.
А сам AdGuard постоянно блокируется :( Если очень нужна блокировка от рекламы, лучше я настрою на самом устройстве (компьютере/телефоне).
